script/content.js

```diff
@@ -0,0 +1,93 @@
+const grid = document.querySelector('.grid')
+const gameContainer = document.querySelector('.game-container') // Select the container
+const timerDisplay = document.createElement('h2')
+timerDisplay.textContent = '⏳ Time: 0 sec'
+gameContainer.insertBefore(timerDisplay, grid) // Insert inside game container
+
+// Select the existing Restart Button from HTML
+const restartButton = document.getElementById('button')
+
+const cardValues = ['🍎', '🍌', '🍒', '🍇', '🍉', '🍋', '🥝', '🍍']
+let gameCards = []
+let flippedCards = []
+let matchedPairs = 0
+let timer
+let secondsElapsed = 0
+let gameStarted = false
+
+// Function to Start/Restart Game
+function setupGame () {
+  grid.innerHTML = ''// Clear old cards
+  gameCards = [...cardValues, ...cardValues].sort(() => Math.random() - 0.5)
+  flippedCards = []
+  matchedPairs = 0
+  secondsElapsed = 0
+  gameStarted = false
+  timerDisplay.textContent = '⏳ Time: 0 sec'
```

Comment:
check to ensure your timer text is properly displayed. It encroaches on the image section